cfn-resources/stream-instance/mongodb-atlas-streaminstance.json

```diff
@@ -0,0 +1,245 @@
+{
+  "typeName": "MongoDB::Atlas::StreamInstance",
+  "description": "Returns, adds, edits, and removes stream instances.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+  "definitions": {
+    "BaseStreamsRegion": {
+      "type": "string",
+      "description": "Name of the cloud provider region hosting Atlas Stream Processing.",
+      "enum": [
+        "VIRGINIA_USA",
+        "CALIFORNIA_USA",
+        "SINGAPORE_SGP",
+        "DUBLIN_IRL",
```

Comment:
Having these enum validations in CFN/TF side can be risky as they usually evolve over time and force us to update frequently. As long as the error message is clear from the API when incorrect value is passed there is no benefit to having these validations. Similar logic applies to other validations defined in this schema.

Reply:
Removed all the hardcoded enum values in the schema